### DISCUSSION TITLE
Broadlink Configuration Node Idea - DHCP related

### DISCUSSION THREAD

**Discussion by eddee54455** [mlfunston/node-red-contrib-broadlink-control#84] [Ideas]
Good day all, and particularly to the authors of this helpful code!!

A suggestion:

Rather than have a separate configuration node, would it be possible to have a "stand-alone" type version, ie mac and IP are captured in the individual nodes(and set capable via a payload possibly) ,  particularly the RM node (I have no experience with the other nodes so I cant comment), due to the following reason: 

In my instance, my dhcp server is a very basic unit without the ability to map mac to IP under DHCP... 

When the  unit router resets and assigns a new IP to the RM, or the RM DHCP lease expires and renewes, I have to manually recode the routine... I can, obviously, scan the network using the "Discover" node and track my RM unit down...

It would be fantastic to be able to use the info from the "Discover" node to resume comms with the RM unit, allowing the RM to be beaten into submission on the fly... LOL....

Possibly there is a way of doing this with the current versions, maybe I have missed it - If I have, please do not hesitate to mention!!

Many thanks
Ed

**Comment by Jibun-no-Kage** [comment]
Really good idea, to not have the IP or MAC address set as static.  You could just add msg.ip and msg.mac to the inbound message for example.
